docs/tool_jdmpview.md

```diff
@@ -36,16 +36,20 @@ The dump viewer is useful for diagnosing `OutOfMemoryError` exceptions in Java&t
 
 ### Starting the dump viewer
 
-`jdmpview (-core <core file> | -zip <zip file>) [-notemp]`
+`jdmpview [-J<vm option>] (-core <core file> | -zip <zip file>) [-notemp]`
 
 
 |          Input option   |  Explanation                                                                                           |
 |-------------------------|--------------------------------------------------------------------------------------------------------|
 | `-core <core file>`     | Specifies a dump file.                                                                                 |
-| `-zip <zip file>`       | Specifies a compressed file containing the core file and associated XML file (produced by the [dump extractor (`jextract`)](tool_jextract.md) tool on AIX&reg;, Linux&reg;, and macOS&reg; systems). |
+| `-zip <zip file>`       | Specifies a compressed file containing the core file (produced by the [dump extractor (`jextract`)](tool_jextract.md) tool on AIX&reg;, Linux&reg;, and macOS&reg; systems). |
 | `-notemp`               | By default, when you specify a file by using the `-zip` option, the contents are extracted to a temporary directory before processing. Use the `-notemp` option to prevent this extraction step, and run all subsequent commands in memory. |
+| `-J-Dcom.ibm.j9ddr.path.mapping=<mappings>` | `<mappings>` is a `:` separated list of native library mappings, each of the form `old-path=new-path`. |
```

Comment:
Please can we say:

The variable `<mappings>` is a list of native library mappings of the form `old-path=new-path`, separated by a colon (`:`).

The variable `<path>` is a list of paths to search for native libraries, separated by a colon (`:`).

Reply:
Updated